podcasts/ProfileViewController.swift

```diff
@@ -322,7 +341,7 @@ class ProfileViewController: PCViewController, UITableViewDataSource, UITableVie
 
     func tableView(_ tableView: UITableView, shouldHighlightRowAt indexPath: IndexPath) -> Bool {
         let row = tableData[indexPath.section][indexPath.row]
-        return row != .kidsProfile
+        return row != .kidsProfile || row != .informationalBanner
```

Comment:
I think this logic may be off. Is it supposed to be `&&` here? I can still highlight the row:

<img src="https://github.com/user-attachments/assets/283002d5-2962-4e02-a023-ac838fcdbdd0" width="300" alt="CleanShot 2025-04-11 at 08 49 50">

```suggestion
        return row != .kidsProfile && row != .informationalBanner
```